galasa-parent/build.gradle

```diff
@@ -34,7 +38,7 @@ allprojects {
         gradlePluginPortal()
         mavenLocal()
         maven {
-            url "https://development.galasa.dev/main/maven-repo/maven/"
+            url "https://development.galasa.dev/gh/maven-repo/maven/"
```

Comment:
This is a tricky one - as we don't want to affect our current Tekton builds, which will be affected by this change. Can you add a second url in here so that both are available?
```
maven {
            url "https://development.galasa.dev/main/maven-repo/maven/"
}
maven {
            url "https://development.galasa.dev/gh/maven-repo/maven/"
}
```

Reply:
done